pandas/core/generic.py

```diff
@@ -4918,7 +4922,12 @@ def describe_categorical_1d(data):
                     names += ['top', 'freq']
                     result += [top, freq]
 
-            return pd.Series(result, index=names, name=data.name)
+            if isinstance(self, ABCSeries):
+                return self._constructor(
+                    result, index=names, name=data.name)
+            else:
```

Comment:
same here (e.g. I can buy ``._constructor(...)``), why do you need the other branch